source/_integrations/androidtv_remote.markdown

```diff
@@ -495,3 +495,4 @@ cards:
 - Some devices, like Xiaomi, become unavailable after they are turned off and can't be turned on with this integration.
 - Some devices experience disconnects every 15 seconds. This is typically resolved by rebooting the Android TV device after the initial setup of the integration.
 - If you are not able to connect to the Android TV device, or asked to pair it again and again, try force stopping the Android TV Remote Service and clearing its storage. On the Android TV device, go to settings, apps, show system apps, select Android TV Remote Service, storage, clear storage. You will have to pair again.
+- Some onscreen keyboards enabled by TV manufactures are known not to support concurrent virtual and onscreen keyboard use. This presents whenever a text field is selected such as "search" where a constant "use the keyboard on your mobile device" will show preventing you from opening the onscreen keyboard to type. This can be overcome by disabling your "3rd party" keyboard and using the default Gboard keyboard.
```

Comment:
```suggestion
- Some onscreen keyboards enabled by TV manufacturers are known not to support concurrent virtual and onscreen keyboard use. This presents whenever a text field is selected, such as "search" where a constant "use the keyboard on your mobile device" will show, preventing you from opening the onscreen keyboard to type. This can be overcome by disabling your "3rd party" keyboard and using the default Gboard keyboard.
```

Reply:
Thanks for fixing! I copied this from the last comment in https://github.com/home-assistant/core/issues/94063 and I missed the typo.